cmd/minikube/cmd/status.go

```diff
@@ -35,13 +38,20 @@ import (
 )
 
 var statusFormat string
+var output string
+
+type KubeconfigStatus struct {
```

Comment:
Could this be a private struct?

If not, please document the use of this struct and its fields.